app/assets/javascripts/d3/mekko.coffee

```diff
@@ -290,20 +298,6 @@ D3.mekko =
                      .text(word)
       )
 
-    # Increase the height of the SVG element to fit the largest unwrapped label.
-    fitHeightToLabels: ->
```

Comment:
Well, it doesn't work with the `empty` combination. When `is_empty` switches from `true` to `false`, the height turns to 0 - which makes sense in the order the methods are executed. I thought this would no longer be needed but I guess it is. I'll see what I can do to make this work.